Fitness/Controllers/OnboardingGymsViewController.swift

```diff
@@ -42,6 +46,8 @@ class OnboardingGymsViewController: UIViewController, UITableViewDelegate, UITab
     // Gyms
     var gymNames: [String] = []
     var favoriteGyms: [String] = []
```

Comment:
would be good to make these vars `private` as well

Reply:
Sounds good! Going to make an issue for the refactor period to make better use of `private` attributes/functions throughout the codebase!